src/components/Desk.tsx

```diff
@@ -4,10 +4,15 @@ import { LetterInteractionData, LetterInterface } from "../types";
 import { Letter } from "./Letter";
 import { ShuffleIcon, ResetIcon, ViewGridIcon } from "@radix-ui/react-icons";
 import { connectDoc } from "../utils/yjs";
-import { useContext } from "react";
+import { useContext, useState } from "react";
 import { UserLetterContext } from "../context/UserLetterContext";
 
-export function Desk() {
+interface Props {
+  highest: number;
+  bumpHighest: () => void;
+}
```

Comment:
do u want to put these in the `UserLetterContext` rather than passing all the way through, and also would name better `highestZIndex` `bumpHighestZIndex()`